src/common/detect-omid.js

```diff
@@ -66,6 +67,7 @@ exports.appendPresenceIframe_ = function(globalObject) {
   iframe.id = exports.OMID_PRESENT_FRAME_NAME;
   iframe.name = exports.OMID_PRESENT_FRAME_NAME;
   iframe.style.display = 'none';
+  iframe.sandbox = undefined;
```

Comment:
FYI setting this to undefined causes the browser to complain; setting it to empty string seems to be the right way to set it to just "sandbox".

Reply:
Oh interesting... Thanks for the follow up. Want me to make this change, or will you do it in the other repo?

Reply:
I did it in the other repo PR. Thanks.